### PR TITLE
Update VC issuer documentation

### PR DESCRIPTION
**Pull request by przydatek**
Add info about building, deploying, and provisioning of the demo VC issuer.

### CODE REVIEW

**Review by frederikrothenberger — APPROVED** [review]
Thanks, left two minor suggestions.

File: demos/vc_issuer/README.md [hunk 1]
```diff
@@ -6,10 +6,20 @@ the [attribute sharing flow on the IC](https://github.com/dfinity/wg-identity-au
 ## Supported Behavior
 
 The app implements the interface of the issuer from [the spec](../../docs/vc-spec.md), plus some additional
-APIs for configuring and testing, cf. [vc_demo_issuer.did](./vc_demo_issuer.did).
-Please note that this issuer is for demonstrating the use of issuer APIS, and is **not meant
+APIs for configuring and testing, cf. [vc_demo_issuer.did](./vc_demo_issuer.did). These additional APIs
+enable "self-registration" of users, so that the issuer can issue credentials for them.
+The issuer also offers a simple FE component, that allows for "self-registration" via a browser.
```
[the remark this answers is not in the document]
Comment (przydatek) [reply demos/vc_issuer/README.md:11]: Added instructions to `HACKING.md`.

File: demos/vc_issuer/README.md [hunk 2]
```diff
@@ -6,10 +6,20 @@ the [attribute sharing flow on the IC](https://github.com/dfinity/wg-identity-au
...
+
+**WARNING:** Please note that this issuer is for demonstrating the use of issuer APIs, and is **not meant
 for real-world deployment**, as it does not have proper management of user data.
 
-## Development
+## Development and Testing
 
 Run `./build.sh` script to build the issuer canister.
+
+An up-to-date binary of the issuer (`vc_demo_issuer.wasm.gz`) is built during the release
```
[the remark this answers is not in the document]
Comment (przydatek) [reply demos/vc_issuer/README.md:20]: ditto

**Review by przydatek — COMMENTED** [review]
Thanks!

**Pull request merged**